cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -388,6 +388,41 @@ func TestGetAPIServerCommand(t *testing.T) {
 				"--etcd-keyfile=faz",
 			},
 		},
+		{
+                        cfg: &kubeadmapi.MasterConfiguration{
+                                API:               kubeadmapi.API{BindPort: 123, AdvertiseAddress: "2001:db8::1"},
+                                Networking:        kubeadmapi.Networking{ServiceSubnet: "bar"},
+                                CertificatesDir:   testCertsDir,
+                                KubernetesVersion: "v1.9.0-beta.0",
+                        },
+                        expected: []string{
+                                "kube-apiserver",
+                                "--insecure-port=0",
+                                "--admission-control=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority,ResourceQuota",
```

Comment:
expect the PersistentVolumeLabel controller in this list; as we do enable the GCE cloud provider.